### PR TITLE
Feature: Event info page-Render Suscribed text when user is logged in

### PR DESCRIPTION
**Pull request by SilviaTrujillano23**
## Describe your changes
 
- Add AuthContext and Provider to store user data
- Render event description from endpoint 
- Remove unused code

## Issue ticket number and link
https://trello.com/c/rHezN0Wq

## Checklist before requesting a review
- [x] I have performed a self-review of my code
- [ ] If it is a core feature, I have added thorough tests.
- [ ] Do we need to implement analytics?
- [ x] Will this be part of a product update? If yes, please write one phrase about this update: Store userdata in authContext
- [ ] Create storybook for components involved

Event info page when user is logged in

image

### CODE REVIEW

File: src/contexts/Auth/AuthProvider.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import React, { useReducer } from 'react'
```
Comment (jose-sal-y-rosas) [inline src/contexts/Auth/AuthProvider.js:1]: Please could you explain why this file is not typescript?

File: src/contexts/Auth/AuthReducer.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+export const reducer = (state, action) => {
```
Comment (jose-sal-y-rosas) [inline src/contexts/Auth/AuthReducer.js:1]: Please could you describe why this file is not typescript?

**Review by jose-sal-y-rosas — CHANGES_REQUESTED** [review]
Please could you describe why yarn.lock is part of the PR? package.json was not modified.

File: src/contexts/Auth/AuthProvider.js (hunk 1 above)
Comment (SilviaTrujillano23) [reply src/contexts/Auth/AuthProvider.js]: My bad, now it is typed.

File: src/contexts/Auth/AuthReducer.js (hunk 2 above)
Comment (SilviaTrujillano23) [reply src/contexts/Auth/AuthReducer.js]: My bad, now it is typed.

**Review by jose-sal-y-rosas — APPROVED** [review]
👍🏼

**Review by jlora-mck — APPROVED** [review]

**Review by FelipeLosada1996 — APPROVED** [review]

**Pull request merged**